ozaria/site/components/teacher-dashboard/modals/common/ButtonImportClassroom.vue

```diff
@@ -0,0 +1,98 @@
+
+<script>
+export default Vue.extend({
+  props: {
+    inactive: {
+      type: Boolean,
+      default: false
+    },
+    inProgress: {
+      type: Boolean,
+      default: false
+    },
+    text: {
+      type: String,
+      required: true,
+      default: 'Link Google Classroom'
+    },
+    inactiveMessage: {
+      type: String,
+      required: false,
+      default: 'Disabled'
+    },
+    iconSrc: {
+      type: String,
+      required: true
+    },
+    iconSrcInactive: {
+      type: String,
+      required: true
+    }
+  },
+  methods: {
+    onClick () {
+      if (!this.inactive && !this.inProgress) {
+        this.$emit('click')
+      }
+    }
+  }
+})
+</script>
+
+<template>
+  <div class="google-classroom-button">
+    <div
+      v-tooltip.bottom="{
+        content: inactive ? inactiveMessage : null
+      }"
+      class="link-google-classroom"
+      :class="{ disabled: inactive || inProgress }"
+      @click="onClick"
+    >
+      <img
+        v-if="inactive"
+        :src="iconSrcInactive"
+      >
+      <img
+        v-else
+        :src="iconSrc"
+      >
+      <span class="google-classroom-text"> {{ text }} </span>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+@import "app/styles/ozaria/_ozaria-style-params.scss";
+.google-classroom-button {
```

Comment:
at first, I thought, u refactored google button to its own cmpt but then it made sense to also use it for ozaria. Seems like we just didn't edit the CSS classes 😂